src/renderer/features/network/NetworkSelector/ui/NetworkSelector.tsx

```diff
@@ -143,7 +113,7 @@ export const NetworkSelector = ({
                                 onChangeCustomNode(node);
                               }}
                             />
-                            {activeNode?.url !== node.url && (
+                            {selectedConnection?.node?.url !== node.url && (
```

Comment:
Let's add this comparison into utils `function canDeleteNode(...)`
So the UI won't bother comparing one url with another